.github/pull_request_template.md

```diff
@@ -1,3 +1,28 @@
+<!--
 
+Unless this is a very simple 1-line-of-code change, please create a new issue describing the change you're proposing first, then link to it from this PR.
+
+Read more about our process in our contributing guide: https://docs.winglang.io/contributors/pull_requests
+
+-->
+
+### Related issue
+
+Fixes # <!-- INSERT ISSUE NUMBER -->
+
+### Description
+
+In plain English, describe your approach to addressing the issue linked above. For example, if you made a particular design decision, let us know why you chose this path instead of another solution.
+
+### Checklist
+
+- [ ] I have updated the PR title to match [Winglang's style guide](https://docs.winglang.io/contributors/pull_requests#how-are-pull-request-titles-formatted)
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation if applicable
+- [ ] I have added tests that prove my fix is effective or that my feature works if applicable
+- [ ] My code changes how tests are executed or modifies code that might be platform-sensitive (related to paths, etc.), so I would like the label `pr/e2e-full` to be added to run tests on all platforms
+
+<!-- If this is still a work in progress, feel free to open a draft PR until you're able to check off all the items on the list above -->
```

Comment:
Should we add a misc section?

Reply:
I removed a lot of the extra text per some feedback